### PR TITLE
🚨🚨🐛 Source Pinterest: Update date-time fields with airbyte_type: timestamp_without_timezone

### PR DESCRIPTION
**Pull request by tolik0**
## What

The format of timestamp fields has been updated to include `"airbyte_type": "timestamp_without_timezone"`. This modification aligns the timestamp fields with Airbyte's data standards.

## 🚨 User Impact 🚨

This PR introduces breaking changes due to changing field type by adding `airbyte_type` for timestamp fields. Users will have to refresh their schemas and refresh data after the upgrade.

### CODE REVIEW

File: airbyte-integrations/connectors/source-pinterest/metadata.yaml [hunk 1]
```diff
@@ -19,6 +19,14 @@ data:
     oss:
       enabled: true
   releaseStage: generally_available
+  releases:
+    breakingChanges:
+      1.0.0:
+        message:
+          This release updates the date-time fields to use the Airbyte format `timestamp_without_timezone`.
+          This change affects all streams where date-time fields are present, ensuring more accurate and standardized time representations.
+          Users will need to refresh the source schema and reset affected streams after upgrading.
```
Comment (katmarkham) [inline airbyte-integrations/connectors/source-pinterest/metadata.yaml:28]: and reset streams BoardPins, BoardSectionPins, Boards, Catalogs, and CatalogFeeds after upgrading.

File: docs/integrations/sources/pinterest-migrations.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+# Pinterest Migration Guide
+
+## Upgrading to 1.0.0
+
+This release updates date-time fields with airbyte_type: timestamp_without_timezone for streams BoardPins, BoardSectionPins, Boards, Catalogs, CatalogFeeds.
+
+Users should:
```
Comment (katmarkham) [inline docs/integrations/sources/pinterest-migrations.md:7]: To ensure uninterrupted syncs, users should:
- Refresh the source schema
- Reset affected streams

**Review by katmarkham — CHANGES_REQUESTED** [review]

File: docs/integrations/sources/pinterest-migrations.md (hunk 2 above)
Comment (tolik0) [reply docs/integrations/sources/pinterest-migrations.md]: Fixed

File: airbyte-integrations/connectors/source-pinterest/metadata.yaml (hunk 1 above)
Comment (katmarkham) [reply airbyte-integrations/connectors/source-pinterest/metadata.yaml:28]: This release updates the date-time fields to use the Airbyte format `timestamp_without_timezone`.
          This change affects all streams where date-time fields are present, ensuring more accurate and standardized time representations: BoardPins, BoardSectionPins, Boards, Catalogs, and CatalogFeeds.
          Users will need to refresh the source schema and reset affected streams after upgrading.

Comment (tolik0) [reply airbyte-integrations/connectors/source-pinterest/metadata.yaml]: Fixed

**Review by bazarnov — APPROVED** [review]

**Review by katmarkham — APPROVED** [review]

**Pull request merged**